src/picongpu/include/simulationControl/MySimulation.hpp

```diff
@@ -423,6 +437,13 @@ class MySimulation : public SimulationHelper<simDim>
             log<picLog::SIMULATION_STATE > ("slide in step %1%") % currentStep;
             resetAll(currentStep);
             initialiserController->slide(currentStep);
+
+            /** add background fields for particle pusher */
+            namespace nvfct = PMacc::nvidia::functors;
+            (*pushBGField)( fieldE, nvfct::Add(), fieldBackgroundE(fieldE->getUnit()),
+                            currentStep, fieldBackgroundE::InfluenceParticlePusher);
+            (*pushBGField)( fieldB, nvfct::Add(), fieldBackgroundB(fieldB->getUnit()),
+                            currentStep, fieldBackgroundB::InfluenceParticlePusher);
```

Comment:
to round it up: `+1` is correct but the movingWindow should be refactored giving a deterministic [getWindow](https://github.com/ax3l/picongpu/blob/e2b5ff3c3a336ca9667b35d69615e0f4fd88d37f/src/picongpu/include/fields/background/cellwiseOperation.hpp#L100-L105) (instead of `numSlides * y`) for every time in the simulation.